config/locales/en.yml

```diff
@@ -1181,6 +1181,7 @@ en:
     already_ordered:
       cart: "cart"
       message_html: "You have an order for this order cycle already. Check the %{cart} to see the items you ordered before. You can also cancel items as long as the order cycle is open."
+    failed: "The checkout failed. Please let us know so that we can process your order."
```

Comment:
I wonder if this is the best message. From a user perspective: who is "us" and do I contact this "us"?

Reply:
I know, it's ambiguous. Notifying the shop owner is probably the most important thing to sort out the order. We can see the error in Bugsnag but another prompt with details would be nice as well. It's difficult to give specific instructions if we don't know what the error is.

In the context of the shop, I hope that "us" is seen as the distributor and that people click on the Contact tab. I will ask our instance managers what they think.

Reply:
in the meantime, I'm pretty sure they know the distributor and how to reach him out (facebook, twitter, website, phone number, etc.). I'd go with this and improve it later if we need to.